ui/src/components/xds-mode-notification.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+
+import { useXdsMode } from "@/hooks/use-xds-mode";
+
+export function XdsModeNotification() {
+  const xds = useXdsMode();
+
+  if (!xds) return null;
+
+  return (
+    <div className="bg-yellow-500 text-center p-2 text-sm">
```

Comment:
nit: `bg-destructive` is defined by shadcn (I don't have a preference personally on yellow vs red:))